clang/test/CodeGenSYCL/kernel-param-acc-array-ih.cpp

```diff
@@ -8,8 +8,7 @@
 
 // CHECK: class kernel_A;
 
-// CHECK: __SYCL_INLINE_NAMESPACE(cl) {
-// CHECK-NEXT: namespace sycl {
+// CHECK: __SYCL_OPEN_NS() {
```

Comment:
There must be check for __SYCL_CLOSE_NS() after the checks for `kernel_names`, `kernel_signatures` and `KernelInfo`.
The namespace can't be closed before these definitions.
The same applies to all other similar CodeGenSYCL tests.

Reply:
I think this should be done separately.

Reply:
I'm okay with that. Feel free to open an issue and notify front-end about that issue.

Reply:
Created - https://github.com/intel/llvm/issues/4437